src/mtev_rest.c

```diff
@@ -196,16 +200,27 @@ mtev_http_get_handler(mtev_http_rest_closure_t *restc) {
     }
     eob--;
   }
+
+  /* no base, give up */
   if(!cont) return NULL;
+
+  const char *protocol = NULL;
+  mtev_hash_table *headers = mtev_http_request_headers_table(req);
+
```

Comment:
stylistically, I think we've been putting local variable definitions in C code directly after open braces.